python-avd/pyavd/_eos_designs/schema/schema_fragments/network_ports.schema.yml

```
@@ -20,6 +20,18 @@ keys:
             The regular expression must match the full hostname.
           items:
             type: str
+        platforms:
+          type: list
```

Let's add `min_length: 1`

and add a comment `# TODO: AVD 6.0.0 add min_length: 1 for switches` 

```suggestion
        platforms:
          type: list
          min_length: 1
```